recipes/r-purrr/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-purrr
+  # Note that conda versions cannot contain -, so any -'s in the version have
+  # been replaced with _'s.
+  version: "0.2.2"
+
+source:
+  fn: purrr_0.2.2.tar.gz
+  url:
+    - http://cran.r-project.org/src/contrib/purrr_0.2.2.tar.gz
+    - http://cran.r-project.org/src/contrib/Archive/purrr/purrr_0.2.2.tar.gz
+  md5: 2277bd639557a38be45ec5d7ba455bbb
+
+
+  # You can add a hash for the file here, like md5 or sha1
+  # md5: 49448ba4863157652311cc5ea4fea3ea
+  # sha1: 3bcfbee008276084cbb37a2b453963c61176a322
+  # patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: testthat, covr
+requirements:
+  build:
+    - r-base
+    - r-bh
+    - r-rcpp
+    - r-dplyr >=0.4.3
+    - r-lazyeval >=0.2.0
+    - r-magrittr >=1.5
+    - posix               # [win]
+    - {{native}}toolchain # [win]
+    - gcc                 # [not win]
```

Comment:
For compiled package we have currently some extra guidlines: https://github.com/bioconda/bioconda-recipes/blob/master/GUIDELINES.md#cc